map.js

```diff
@@ -11,70 +17,86 @@ let mapY = 0;
 
 let mousedown = false;
 
-let debug = true; //makes iframe and guide small by default for map function testing.
+let img_Map;
+
+let icon_Ayleid;
+let icon_Camp;
+let icon_Cave;
+let icon_Fort;
+let icon_Gate;
+let icon_Inn;
+let icon_Settlement;
+let icon_Mine; //still needs image
+let icon_Landmark; //still needs image
+let icon_Shrine; //still needs image
+
+let locArr;
+
+let debug = false; //makes iframe and guide small by default for map function testing.
 
 function initMap(){
+    //load map cord data
+    fetch("data/locations.json").then(response => response.json()).then(response => locArr = response.elements);
+
     canvas = document.getElementById("canvas_Map");
     mapCanvasContext = canvas.getContext("2d");
     
-    img = document.createElement("img");
-    img.width = 3544;
-    img.height = 2895;
-    img.src = "images/Cyrodil_Upscaled.png";
-    img.onload = drawMap();
-
+    initImgs();
+        
     //center map on imp city
-    mapX = img.width/2.1;
-    mapY = img.height/3.2;
+    mapX = img_Map.width/2.1;
+    mapY = img_Map.height/3.2;
```

Comment:
if we're centering on imperial city, might as well specify exact coords?

Reply:
done